src/app/desktop/desktop-startmenu/desktop-startmenu.component.scss

```diff
@@ -185,6 +185,10 @@ $taskbar-height: 5.7%;
         height: auto;
         margin-right: 9px;
       }
+
+      .cc {
+        filter: invert(100%) sepia(100%) saturate(0%) hue-rotate(200deg) brightness(101%) contrast(103%);
```

Comment:
Change the color in the svg file itself and don't use filters.

Reply:
done